user/timepix3/module/src/Timepix3Event2StdEventConverter.cc

```diff
@@ -197,13 +234,56 @@ bool Timepix3RawEvent2StdEventConverter::Converting(eudaq::EventSPC ev, eudaq::S
       // Convert final timestamp into picoseconds
       const uint64_t timestamp = time * 1000 / 4096 * 25;
 
-      // Set event start/stop:
-      event_begin = (timestamp < event_begin) ? timestamp : event_begin;
-      event_end = (timestamp > event_end) ? timestamp : event_end;
-
-      // creating new pixel object with non-calibrated values of tot and toa
-      plane.PushPixel(col, row, tot, timestamp);
-    }
+      // Apply calibration if applyCalibration is true
+      // (copied over from Corryvreckan EventLoaderTimepix3)
+      if(applyCalibration) {
+        // EUDAQ_DEBUG()"Applying calibration to DUT"); // cannot change verbosity in Corryvreckan
```

Comment:
Remove completely? Or might there be a solution in the future?

Reply:
Not unless you cook one up. In principle we should have access to the EUDAQ logger, but you'd have to dig around and see how levels are changed

Reply:
Then I'll just leave it in and if we tackle this in the future, we know where the DEBUG outputs should be.